### PR TITLE
BS-13818 Support multiple composed type fields

### PR DESCRIPTION
**Pull request by rbioteau**
Support multiple composed reference for operation generation

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/BusinessObjectListInitializer.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        return true;
+    }
+
+    private void returnListVar(final StringBuilder scriptBuilder, final BusinessObject businessObject) {
+        scriptBuilder.append("return");
+        scriptBuilder.append(" ");
+        scriptBuilder.append(listVarName(businessObject));
+    }
+
+    private void forEach(final StringBuilder scriptBuilder, final BusinessObject businessObject) throws BusinessObjectInstantiationException {
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/BusinessObjectListInitializer.java]: would be nice to add in javadoc a sample of the result of what is geenrated, it is hard to guess by reading code.

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/BusinessObjectListInitializer.java]: What about unit tests ?

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/BusinessObjectListInitializer.java]: there is no unit test for this method ;precisely as it is a private one

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/BusinessObjectListInitializer.java]: Added some comment to ease the reading of the code

**Pull request merged**